people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Team
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
```

Comment:
I'm flattered to still be included here, but realistically I don't consider myself part of the core pandas team any more (feel free to withdraw my commit rights). I'm still quite happy to be pinged if there's something I can particularly help with, such as IPython/Jupyter related issues, but I haven't regularly contributed to the project for some time.

:+1: to this overall idea; I haven't read through all the documents here, but I was happy with it when we wrote it for Jupyter.